src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable {
+    private double radius = Math.random() * 10;
```

Comment:
Do not set random field values directly, pass them through the constructor

Reply:
done